pkg/controllers/resources/endpoints/syncer.go

```diff
@@ -41,79 +36,23 @@ func (s *endpointsSyncer) Sync(ctx *synccontext.SyncContext, pObj client.Object,
 var _ syncer.Starter = &endpointsSyncer{}
 
 func (s *endpointsSyncer) ReconcileStart(ctx *synccontext.SyncContext, req ctrl.Request) (bool, error) {
```

Comment:
I think we are missing a skip (`return true, nil`) of the "kubernetes" endpoints resource in the "default" namespace. You, know the one we create with the "k8sdefaultendpoint" controller.